.github/workflows/imagepuller-benchmark.yml

```diff
@@ -44,15 +39,8 @@ jobs:
           [registries."ghcr.io."]
           auth = "$auth"
           EOF
-      - name: Download baseline JSON from S3
-        run: |
-          aws s3 cp "s3://contrast-public/imagepuller-benchmark/baseline.json" "baseline.json"
       - name: Run imagepuller-benchmark script
         run: |
-          imagepullereval="$(nix build .#imagepuller-benchmark --print-out-paths)/bin/imagepuller-benchmark"
+          imagpullerbenchmark="$(nix build .#imagepuller-benchmark --print-out-paths)/bin/imagepuller-benchmark"
           imagepuller="$(nix build .#imagepuller --print-out-paths)/bin/imagepuller"
-          sudo "$imagepullereval" "$imagepuller" -c baseline.json -o baseline-new.json -i tools/imagepuller-benchmark/images.txt -a imagepuller.toml
-      - name: Upload new baseline.json to S3
-        if: github.event_name == 'push' && github.ref_name == 'main'
-        run: |
-          aws s3 cp "baseline-new.json" "s3://contrast-public/imagepuller-benchmark/baseline.json"
+          sudo "$imagpullerbenchmark" "$imagepuller" -l tools/imagepuller-benchmark/limits.json -o baseline-new.json -i tools/imagepuller-benchmark/images.txt -a imagepuller.toml
```

Comment:
I think we can remove `-o baseline-new.json` here and in the go code?

Reply:
I think I'd like to keep it in the go code, to have a quick and easy way to generate a limits file. But yes, that it's still in the yml is an oversight.